snippets/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "dependencies": {
-        "http-server": "^0.11.1"
+        "http-server": "^0.11.1",
+        "markdownlint": "^0.17.2"
```

Comment:
This seems to be a mistake?

Reply:
I had to install markdownlint in order to check the linting. the cloud function was failing if not...

Reply:
Oh sorry you are right. I misread which package.json it was